src/smriprep/interfaces/fsl.py

```diff
@@ -0,0 +1,26 @@
+from nipype.interfaces.base import traits
+from nipype.interfaces.fsl.preprocess import FAST, FASTInputSpec
+
+
+class _FixTraitFASTInputSpec(FASTInputSpec):
+    bias_iters = traits.Range(
+        low=0,
+        high=10,
+        argstr='-I %d',
+        desc='number of main-loop iterations during bias-field removal',
+    )
+
+
+class FixBiasItersFAST(FAST):
```

Comment:
Since this is supported by FAST, no need to call it anything else. We should probably backport this to nipype at some point...
```suggestion
class FAST(_FAST):
```